### DISCUSSION TITLE
qterm is not running in the terminal

### DISCUSSION THREAD

**Discussion by Aur0nd** [aws/codewhisperer-command-line-discussions#196] [Support ticket]
### Checks

- [X] I ran `q doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `q restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/q-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

q doctor not passing checks

### Actual behaviour

<img width="1042" alt="Screenshot 2024-06-03 at 11 49 46" src="https://github.com/aws/q-command-line-discussions/assets/71910902/13ad9e1c-2aa0-4341-aabe-6c6f8d0d71e4">

### Steps to reproduce

- ssh integration works in both host and remote
- I've restarted both host and remote
- q doctor works in host

### Environment

```toml
%5Bq-details%5D%0Aversion %3D │%221.1.0%22%0Ahash %3D %22arn%3Aaws%3As3%3A%3A%3Afig-io-desktop-pipeline-artifacts-158872659206-us-west-2%2FFigIoDeskt│op-Pipelin%2FfigIoDeskt%2FuKWARoV%22%0Adate %3D %222024-05-21T20%3A19%3A55.085016Z %2813d ago%29%22%0A%0A%5Bsystem-inf│o%5D%0Achip %3D %22Intel%28R%29 Core%28TM%29 i5-6500 CPU %40 3.20GHz%22%0Atotal-cores %3D 4%0Amemory %3D %2231.22 GB%2│2%0A%0A%5Bsystem-info.os.Linux%5D%0Akernel_version %3D %226.2.16-18-pve%22%0A%0A%5Bsystem-info.os.Linux.os_release%5D%│0Aid %3D %22debian%22%0Aname %3D %22%5C%22Debian GNU%2FLinux%5C%22%22%0Aversion_id %3D %22%5C%2212%5C%22%22%0Aversion │%3D %22%5C%2212 %28bookworm%29%5C%22%22%0A%0A%5Benvironment%5D%0Acwd %3D %22%2Fhome%2FUSER%22%0Acli-path %3D %22%2Fhom│e%2FUSER%2F.local%2Fbin%2Fq%22%0Ashell-path %3D %22%2Fusr%2Fbin%2Fbash%22%0Ashell-version %3D %225.2.15%22%0Ainstall-m│ethod %3D %22unknown%22%0A%0A%5Benv-vars%5D%0APATH %3D %22%2Fhome%2FUSER%2F.local%2Fbin%3A%2Fusr%2Flocal%2Fbin%3A%2Fus│r%2Fbin%3A%2Fbin%3A%2Fusr%2Flocal%2Fgames%3A%2Fusr%2Fgames%3A%2Fhome%2FUSER%2F.local%2Fbin%22%0AQ_SET_PARENT_CHECK %3D│ %221%22%0ASHELL %3D %22%2Fbin%2Fbash%22%0ATERM %3D %22xterm-256color%22%0AXDG_SESSION_TYPE %3D %22tty%22%0A
```

**Comment by grant0417** [comment]
Can you run through the following steps to make sure everything is configured correctly:

### On the host:
- Ensure the integration is installed with `q integrations status ssh`

### On the remote:
- Ensure the `sshd_config` includes the lines:
```
AcceptEnv Q_SET_PARENT
AllowStreamLocalForwarding yes
```
- Restart the `sshd` instance and reconnect, likely: `sudo systemctl restart sshd`
- Run `q doctor` to check the remote machine

**Reply by Aur0nd** [reply]
Thanks @grant0417 

I've done all that multiple times, unfortunately I still have the same issue 
<img width="358" alt="Screenshot 2024-06-04 at 06 38 04" src="https://github.com/aws/q-command-line-discussions/assets/71910902/0e94ada8-fffa-4c94-b3dd-c0df9acd1cee">
<img width="999" alt="Screenshot 2024-06-04 at 06 38 35" src="https://github.com/aws/q-command-line-discussions/assets/71910902/bf2e34c9-0c02-4de8-8ca6-4576e2f7cd67">

**Reply by FastBoot3DS** [reply]
> Thanks @grant0417 
> 
> 
> 
> I've done all that multiple times, unfortunately I still have the same issue 
> 
> <img width="358" alt="Screenshot 2024-06-04 at 06 38 04" src="https://github.com/aws/q-command-line-discussions/assets/71910902/0e94ada8-fffa-4c94-b3dd-c0df9acd1cee">
> 
> <img width="999" alt="Screenshot 2024-06-04 at 06 38 35" src="https://github.com/aws/q-command-line-discussions/assets/71910902/bf2e34c9-0c02-4de8-8ca6-4576e2f7cd67">
> 
> 

I also have this same issue can't seem to resolve it, everything worked perfectly with fig and then Amazon went and changed it all.
